docs/source/markdown/podman-create.1.md

```diff
@@ -992,6 +992,19 @@ Maximum time a container is allowed to run before conmon sends it the kill
 signal.  By default containers will run until they exit or are stopped by
 `podman stop`.
 
+#### **--tls-verify**=**true**|**false**
+
+Skip TLS Verification while **podman-create** tries to pull an image from an insecure registry and create a container. The default is **true**.
+
+```
+$ podman create --tls-verify=false localhost:5000/myalpine ls
+```
+
+This command skips TLS verification, pulls **myalpine** image from insecure-registry running on localhost:5000 and creates a container.
+
+When set to **false** TLS verification will be skipped, while dealing with insecure registries. 
+Hence a user won't require HTTPS and  certificates verification when contacting registried for pulling images.
+
```

Comment:
I think that's too much information (and inconsistent with the other man pages). Compare to https://github.com/containers/podman/blob/main/docs/source/markdown/podman-pull.1.md#--tls-verifytruefalse.

Could you copy over the description from the other man page?

Reply:
Done, now it's the same as the other Man pages.